ODBC: unixodbc_setup.sh

tiagokepe opened the pull request:
Hey all,

This PR adds the script to configure ODBC on Linux and also adds it into the Linux ODBC asset file.

tiagokepe committed:
ODBC: first version of unixodbc_setup.sh

tiagokepe committed:
ODBC: including 'unixodbc_setup.sh' to Linux ODBC asset file.

Mytherin commented:
Thanks for the PR!

Perhaps add `set -e` to the top of the script to ensure execution is stopped when an error is encountered?

tiagokepe committed:
ODBC: linux setup with -db and -D options.

tiagokepe committed:
ODBC: using the unixodbc_setup.sh to test the script.

tiagokepe commented:
> Thanks for the PR!
> 
> Perhaps add `set -e` to the top of the script to ensure execution is stopped when an error is encountered?

Sure, last commit added it.
I also added new options to the script the "-db" and "-D", and changed the test script to use the unixodbc_setup.sh with different options.

Mytherin commented:
Thanks! Looks good.
